eng/scripts/bomhelpers.ps1

```diff
@@ -2,310 +2,312 @@
 # Licensed under the MIT License.
 
 class MavenArtifactInfo {
-    [String] $GroupId
-    [String] $ArtifactId
-    [String] $LatestGAOrPatchVersion
-    [String] $LatestRealeasedVersion
+  [String] $GroupId
+  [String] $ArtifactId
+  [String] $LatestGAOrPatchVersion
+  [String] $LatestReleasedVersion
 
-    MavenArtifactInfo($ArtifactId, $LatestGAOrPatchVersion, $LatestRealeasedVersion) {
-        $this.ArtifactId = $ArtifactId
-        $this.LatestGAOrPatchVersion = $LatestGAOrPatchVersion
-        $this.LatestRealeasedVersion = $LatestRealeasedVersion
-        $this.GroupId = 'com.azure'
-    }
+  MavenArtifactInfo($ArtifactId, $LatestGAOrPatchVersion, $LatestReleasedVersion) {
+    $this.ArtifactId = $ArtifactId
+    $this.LatestGAOrPatchVersion = $LatestGAOrPatchVersion
+    $this.LatestReleasedVersion = $LatestReleasedVersion
+    $this.GroupId = 'com.azure'
+  }
 }
 
-. Join-Path ${PSScriptRoot} "common.ps1"
+$RepoRoot = Resolve-Path "${PSScriptRoot}../../.."
+$CommonScriptFilePath = Join-Path $RepoRoot "eng" "common" "scripts" "common.ps1"
+. $CommonScriptFilePath
 
 function SetDependencyVersion($GroupId = "com.azure", $ArtifactId, $Version) {
-    $repoRoot = Resolve-Path "${PSScriptRoot}..\..\.."
-    $setVersionFilePath = Join-Path $repoRoot "eng" "versioning" "set_versions.py"
-    $cmdOutput = python $setVersionFilePath --bt client --new-version $Version --ar $ArtifactId --gi $GroupId
-    $cmdOutput = python $setVersionFilePath --bt client --ar $ArtifactId --gi $GroupId --increment-version
+  $repoRoot = Resolve-Path "${PSScriptRoot}../../.."
+  $setVersionFilePath = Join-Path $repoRoot "eng" "versioning" "set_versions.py"
+  $cmdOutput = python $setVersionFilePath --bt client --new-version $Version --ar $ArtifactId --gi $GroupId
```

Comment:
FYI powershell will not stop on errors from external binaries (at least in the GA versions we are using). If you want that, you can do something like:

```
if ($LASTEXITCODE) {
  exit $LASTEXITCODE
}
```